src/zzz_od/gui/view/devtools/mcp_service_interface.py

```
@@ -0,0 +1,471 @@
+
+        return content
+
+    def on_interface_shown(self) -> None:
+        """页面显示时启动状态/日志轮询，并立即探测一次服务。"""
+        super().on_interface_shown()
+        self._status_timer.start()
+        self._log_timer.start()
+        self._run_action('probe')
+
+    def on_interface_hidden(self) -> None:
+        """页面隐藏时停止轮询，避免后台无意义刷新 UI。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)